pygac/__init__.py

```diff
@@ -52,3 +53,10 @@ def centered_modulus(array, divisor):
     arr = array % divisor
     arr[arr > divisor / 2] -= divisor
     return arr
+
+
+def calculate_sun_earth_distance_correction(jday):
+    """Calculate the sun earth distance correction."""
```

Comment:
Can you please add a note to the docstring that this formula was chosen because it most closely matches PATMOS-x (as explained by @abhaydd on slack)?